### DISCUSSION TITLE
working with aiMaterial

### DISCUSSION THREAD

**Discussion by cj2000** [assimp/assimp#4604] [Q&A] [answered]
Since my previous question was misunderstood, I am asking again. So how to get from aiMaterial such things like difuse, ambiente and specular color, but also names of textures (difuse, bump and so on) .

**Comment by SC5Shout** [comment, marked as answer]
google -> write assimp materials -> find the first link -> open it http://assimp.sourceforge.net/lib_html/materials.html -> write some code
